zuul.d/jobs.yaml

```diff
@@ -354,6 +354,46 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/frr.frr
 
+- job:
+    name: ansible-test-units-frr-python27
+    parent: ansible-test-units-base-python27
+    required-projects:
+      - name: github.com/ansible-collections/frr.frr
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/frr.frr
+
+- job:
+    name: ansible-test-units-frr-python35
+    parent: ansible-test-units-base-python35
+    required-projects:
+      - name: github.com/ansible-collections/frr.frr
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/frr.frr
+
+- job:
+    name: ansible-test-units-frr-python36
+    parent: ansible-test-units-base-python36
+    required-projects:
+      - name: github.com/ansible-collections/frr.frr
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/frr.frr
+
+- job:
+    name: ansible-test-units-frr-python37
+    parent: ansible-test-units-base-python37
+    required-projects:
+      - name: github.com/ansible-collections/frr.frr
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/frr.frr
+
+- job:
+    name: ansible-test-units-frr-python38
+    parent: ansible-test-units-base-python38
+    required-projects:
+      - name: github.com/ansible-collections/frr.frr
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/frr.frr
```

Comment:
we can delete this, we run py38 units from EEs

Reply:
@pabelanger ack. 

https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/jobs.yaml#L349 also seems to runs python 3.8 unit tests, we can remove that too?

Same question for ovs: https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L686

Reply:
Ack. Pushed the changes.